workers/unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -154,8 +154,9 @@ PlayerSettings:
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
-    Android: io.improbable.gdk
+    Android: io.improbable.gdk.starterproject
     Standalone: com.improbable.gdk.starterproject
+    iOS: com.improbable.gdk.starterproject
```

Comment:
why is the Android one `io..` but the Standalone/iOS ones `com...`?

Reply:
yep, that should be the same one for both :)